netbox/dcim/svg/cables.py

```diff
@@ -238,20 +245,39 @@ def draw_fanout(self, node, connector):
             Polyline(points=points, style=f'stroke: #{connector.color}'),
         ))
 
-    def draw_cable(self, cable):
-        labels = [
-            f'Cable {cable}',
-            cable.get_status_display()
-        ]
-        if cable.type:
-            labels.append(cable.get_type_display())
-        if cable.length and cable.length_unit:
-            labels.append(f'{cable.length} {cable.get_length_unit_display()}')
+    def draw_cable(self, cable, terminations, cable_count=0):
```

Comment:
`draw_cable()` should be limited to rendering a single cable. Where multiple cables need to be displayed, it should be called multiple times.

Reply:
It does only draw a single cable, the terminations are passed to get position data on each termination.